src/jquery.gridster.js

```diff
@@ -791,45 +791,6 @@
 
 
 	/**
-	 * Change the dimensions of widgets.
-	 *
-	 * @method resize_widget_dimensions
-	 * @param {Object} [options] An Object with all options you want to
-	 *        overwrite:
-	 *    @param {Array} [options.widget_margins] Margin between widgets.
-	 *     The first index for the horizontal margin (left, right) and
-	 *     the second for the vertical margin (top, bottom).
-	 *    @param {Array} [options.widget_base_dimensions] Base widget dimensions
-	 *     in pixels. The first index for the width and the second for the
-	 *     height.
-	 * @return {Class} Returns the instance of the Gridster Class.
-	 */
-	fn.resize_widget_dimensions = function (options) {
```

Comment:
Because there are two functions with the same name, and the second (the one that I'm fixing in this patch) is overriding the first one.

First function in line [807](https://github.com/dsmorse/gridster.js/blob/master/src/jquery.gridster.js#L807)
Second function in line [4030](https://github.com/dsmorse/gridster.js/blob/master/src/jquery.gridster.js#L4030)